File: CS50x/lecture1/cash/cash.c
#include <stdio.h>
#include <cs50.h>

int main (void)
{
    while (true)
    {
        int a = 0;
        int i = get_int("Change owed: ");
        if (i > 0)
        {
            while (i >= 25)
            {
                a = a + 1;
                i = i - 25;
            }
            while (i >= 10)
            {
                a = a + 1;
                i = i - 10;
            }
            while (i >= 5)
            {
                a = a + 1;
                i = i - 5;
            }
            while (i >= 1)
            {
                a = a + 1;
                i = i - 1;
            }
            printf("%d\n", a);
            break;
        }
    }

}

File: CS50x/lecture6/sentimental-cash/cash.py
while True:
    a = 0
    while True:
        try:
            b = float(input("Change owed: "))
            break
        except ValueError:
            True
    i = b * 100
    if i > 0:
        while i >= 25:
            a = a + 1
            i = i - 25
        while i >= 10:
            a = a + 1
            i = i - 10
        while i >= 5:
            a = a + 1
            i = i - 5
        while i >= 1:
            a = a + 1
            i = i - 5
        print(a)
        break
